samples/snippets/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -1659,21 +1660,23 @@ static void cancelCreateBackup(
   // [END spanner_cancel_backup_create]
 
   // [START spanner_list_backup_operations]
-  static void listBackupOperations(InstanceAdminClient instanceAdminClient, DatabaseId databaseId) {
+  static void listBackupOperations(
```

Comment:
Isn't this a breaking change? Maybe customers won't care because it is in the sample , but I don't know what the policy is here. Yoshi team might know

Reply:
@olavloite pointed out that this is a non public method and hence can't be called so this will not be a breaking change

Reply:
That is true, but we publish these samples in our public documentation, so we would be changing the signature. Anyway, if your team is ok with this, I am ok.